### PR TITLE
# 24 New/bootstrap styles

### PR DESCRIPTION
**Pull request by twistedomelette**
Added global font "Nunito" and colors. Task 24

### CODE REVIEW

File: frontend/src/assets/css/theming.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
+@import "~bootstrap/scss/functions";
+@import "~bootstrap/scss/variables";
+
+/* stylelint-disable */
+
+// $newBlack:    #1d1d1d !default;
```
Comment (nikita-remeslov) [inline frontend/src/assets/css/theming.scss]: Should remove if we don't use it

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
I don't see global classes for font size. Check task description

File: frontend/src/assets/css/theming.scss (hunk 1 above)
Comment (twistedomelette) [reply frontend/src/assets/css/theming.scss]: removed

**Review by nikita-remeslov — APPROVED** [review]
Looks good

**Pull request merged**